code/__DEFINES/research.dm

```diff
@@ -1,4 +1,4 @@
-#define SHEET_MATERIAL_AMOUNT 1
+#define SHEET_MATERIAL_AMOUNT 2000
```

Comment:
Does this only affect the circuits, or does it affect other research?

Reply:
It's affect only circuit, it was commented before me.